sdk/core/azure-core-amqp/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## Version 1.0.0-beta.9 (2020-01-07)
-- Add links on batch send operation in eventhubs
+## 1.0.0 (2020-01-07)
```

Comment:
Shouldn't this be 1.0.1 ?  Because that is what  you have specified in  version_client.txt

Reply:
This is fixed now.